kano_settings/system/audio.py

```diff
@@ -99,13 +99,13 @@ def is_HDMI():
 
     if amixer_string.find(hdmi_string) != -1:
         # Make sure config file is up to date
-        if get_setting('Audio') != 'HDMI':
-            set_setting('Audio', "HDMI")
+        if get_setting('Audio') != _("HDMI"):
+            set_setting('Audio', _("HDMI"))
```

Comment:
A couple of things here: 

1. "Analogue" here is directly written to a config file in the `set_settings` function which I don't know how the rest of the app / system copes with.
2. Given that `_()` returns unicode, if the translated string will contain non-ascii characters, this line will fail without a `.encode('utf-8')`